ndcube/extra_coords/extra_coords.py

```diff
@@ -406,6 +419,105 @@ def dropped_world_dimensions(self):
 
         return dict()
 
+    def interpolate(self, new_array_grids, ndcube=None, **kwargs):
```

Comment:
Does this kwargs need to be here or does it need to be passed into the function call insdie this function?

Reply:
The kwargs should have been passed to the interpolation function used under the hood.  I've made this change so they now are.